### PR TITLE
Export secret values verbatim

### PR DESCRIPTION
**Pull request by tmatilai**
Parse op inject output per variable using a marker line, so values may contain newlines, leading or trailing whitespace, or anything else. Export directly instead of going through direnv dotenv and eval.

Validate the template before calling op and never log op output. Ignore blank lines and comments in all input modes. URL-encode masked values as GitHub Actions requires.

The op stub now passes text through verbatim like the real command. The harness runs .envrc with an empty stdin, since from_op reads any non-TTY stdin. Tests no longer need the real direnv binary.

### CODE REVIEW

File: tests/from_op.bats [hunk 1]
```diff
@@ -19,7 +19,7 @@ run_envrc() {
         source ./tests/stubs.bash
         source ./1password.sh
         source "$2"
-    ' bash "$REPO_ROOT" "$envrc"
+    ' bash "$REPO_ROOT" "$envrc" </dev/null
```
Comment (bobvanderlinden) [inline tests/from_op.bats:22]: Why is this needed? Does some process accept stdin? That shouldn't be happening?

File: 1password.sh [hunk 2]
```diff
@@ -102,71 +102,76 @@ from_op() {
...
+            if [[ $GHA_MASKING -ne 0 && -n $value ]]; then
+                # Mask secret values in GitHub Actions logs. Special characters
+                # must be URL-encoded, `%` first.
+                # See https://docs.github.com/en/actions/reference/workflows-and-actions/workflow-commands#masking-a-value-in-a-log
+                masked="${value//%/%25}"
+                masked="${masked//$'\r'/%0D}"
+                masked="${masked//$'\n'/%0A}"
+                echo "::add-mask::${masked}"
+            fi
+            export "${keys[i]}=$value"
```
Comment (bobvanderlinden) [inline 1password.sh:159]: I'm unsure what the edge-cases of `export` are, but would it make sense to add tests for `'` and `"`?

**Review by bobvanderlinden — DISMISSED** [review]
Ah that's a good one. Avoids having to rely on direnv's parser and requiring escaping for the parser. Makes sense to me :+1:

File: tests/from_op.bats (hunk 1 above)
Comment (tmatilai) [reply tests/from_op.bats:22]: `from_op` so far always tries to read stdin (which will be fixed in #27). So if the caller of these tests manages to have stdin open, this would hang. Maybe not a very common problem, but this came up in an AI assisted audit I ran through the whole repo.

File: 1password.sh (hunk 2 above)
Comment (tmatilai) [reply 1password.sh:159]: The promise is that `export` doesn't process the `$value` in any way. But that's a good idea also for future! I'll push some test case for special characters, in addition for the dollar sign which was added recentyl.

**Review by bobvanderlinden — APPROVED** [review]
Looks good 👍
